src/frontend/staff-portal/src/app/components/staff-workbench/contact-info/contact-info.component.html

```diff
@@ -373,6 +373,8 @@ <h4>Who are we contacting?</h4>
       </mat-expansion-panel>
       <app-ticket-request [disputeInfo]="lastUpdatedDispute"
         *ngIf="lastUpdatedDispute.emailAddressVerified === true || !(lastUpdatedDispute.emailAddress)"></app-ticket-request>
```

Comment:
You can use ng-container to include these three code blocks, so you don't need to repeat the same condition 3 times

Reply:
Done